consul/base.py

```diff
@@ -1856,11 +1886,15 @@ def node(self, node, index=None, wait=None, consistency=None, dc=None):
             *consistency* can be either 'default', 'consistent' or 'stale'. if
             not specified *consistency* will the consistency level this client
             was configured with.
+
+            *token* is an optional `ACL token`_ to apply to this request.
             """
             params = []
             dc = dc or self.agent.dc
             if dc:
                 params.append(('dc', dc))
+            if token:
```

Comment:
```suggestion
            token = token or self.agent.token
            if token:
```
agent may already have token set. Use it!